server/routes/private.js

```diff
@@ -63,18 +63,19 @@ router.post('/api/add-recurring-schedule', async (req, res) => {
 
     const query = req.query ? req.query : {};
 
+    
     const slotId = query.slotId ? query.slotId : null;
-    const schoolId = query.schoolID ? query.schoolID : null;
-    const workerId = query.workerID ? query.workerID : null;
-    const status = query.status ? query.status : null;
+    const schoolId = query.schoolId ? query.schoolId : null;
+    const workerId = query.workerId ? query.workerId : null;
+    //const status = query.status ? query.status : null;
```

Comment:
Hi Peter,
To increase the flexibility of this endpoint for future uses, I would recommend keeping status as one of the parameters that the user can pass in. Instead of storing it as null in the event that it is undefined, use available as the default. That way, the other methods can insert using status. Thus, line 70 would be `const status = query.status ? query.status : TimeslotStatus.available` (or something around that logic).